ui/index.js

```diff
@@ -275,6 +275,21 @@ async function runInitialActions(store) {
   }
 }
 
+export async function getCleanAppState(store) {
+  // here, using `cloneDeep` instead of `clone` to avoid mutating the original nested state.
+  const state = cloneDeep(store.getState());
```

Comment:
Using `cloneDeep` here might be quite memory intensive.
Maybe you can do the following instead?
```ts
const state = { ...store.getState() }
```